tests/integration/api/queries/orderByReferenceId/orderByReferenceId.test.js

```diff
@@ -0,0 +1,108 @@
+import TestApp from "/imports/test-utils/helpers/TestApp";
+import Factory from "/imports/test-utils/helpers/factory";
+import { getAnonymousAccessToken } from "/imports/plugins/core/orders/server/no-meteor/util/anonymousToken";
+
+// Mock data for logged-in user test
+const mockShopId = "integ-test-shop-id";
+const opaqueShopId = "cmVhY3Rpb24vc2hvcDppbnRlZy10ZXN0LXNob3AtaWQ=";
+const shopName = "Test Shop";
+const orderId = "integ-test-order-id";
+const orderReferenceId = "123456789";
+const accountId = "integ-test-account-id";
+const opaqueAccountId = "cmVhY3Rpb24vYWNjb3VudDppbnRlZy10ZXN0LWFjY291bnQtaWQ=";
+
+const mockOrdersAccount = Factory.Accounts.makeOne({
```

Comment:
This should be `Factory.Account` instead of `Accounts`.

Reply:
Fixed, thanks!